README.adoc

```diff
@@ -167,10 +167,13 @@ You can access the application by making requests to the `gateway` job endpoints
 |Get a specific job |GET \http://localhost:8080/api/jobs/{jobId} |`{"jobId":"661891cb-ad36-4ef4-9bb3-641f973f2964","result":5}`
 |===
 
+
 To create a job, you can use `curl -X POST \http://localhost:8080/api/jobs` command if available on the system. The Postman application can also be used. The request take some time for the job results to return.
 
 The completed jobs JSON output with a created job looks like `{"averageResult":5.0,"count":1,"results":[{"jobId":"661891cb-ad36-4ef4-9bb3-641f973f2964","result":5}]}`. The `averageResult` attribute is the average sleep time of all the jobs. The `count` attribute is the number of jobs, and the `results` attribute contains the list of the jobs. The JSON output for each job has a job ID and a sleep time as the result for the job.
 
+If no jobs have been created, the JSON output will be this `{"count":0,"results":[]}`. The `count` attribute is 0 because there are no jobs yet, and the `results` attribute is empty for the same reason.
```

Comment:
Suggested update:
"If no jobs are created, the JSON output is `{"count":0,"results":[]}`. Because there are no jobs yet, the `count` attribute is 0 and the `results` attribute is empty."